frontend/src/features/workspace/components/CreateFormModal/CreateFormWizardProvider.tsx

```diff
@@ -69,6 +72,8 @@ const useCreateFormWizardContext = (): CreateFormWizardContextReturn => {
         title,
         responseMode,
         publicKey: keypair.publicKey,
+        // do not mutate with workspaceId if it is 'All Forms'
+        workspaceId: isDefaultWorkspace ? undefined : activeWorkspace._id,
```

Comment:
can add a comment here to explain why this is necc!

Reply:
[26ce06b](https://github.com/opengovsg/FormSG/pull/6755/commits/26ce06b2d013347652ed5db548c1bfa31415effa)